packages/smart/contracts/turbo/AbstractMarketFactory.sol

```diff
@@ -20,12 +20,27 @@ abstract contract AbstractMarketFactory is TurboShareTokenFactory {
     event SharesBurned(uint256 id, uint256 amount, address receiver);
     event WinningsClaimed(uint256 id, uint256 amount, address indexed receiver);
 
+    event SettlementFeeClaimed(address settlementAddress, uint256 amount, address indexed receiver);
+    event ProtocolFeeClaimed(address protocol, uint256 amount);
+
+    event ProtocolChanged(address protocol);
+    event ProtocolFeeChanged(uint256 fee);
+    event SettlementFeeChanged(uint256 fee);
+    event StakerFeeChanged(uint256 fee);
+
     IERC20Full public collateral;
     FeePot public feePot;
+
+    // fees are out of 1e18 and only apply to new markets
     uint256 public stakerFee;
-    uint256 public creatorFee;
-    // creator address => amount of collateral
-    mapping(address => uint256) public accumulatedCreatorFees;
+    uint256 public settlementFee;
+    uint256 public protocolFee;
+
+    address public protocol; // collects protocol fees
```

Comment:
What is this exactly? Can we rename it to something more descriptive?

Reply:
@bthaile 

https://github.com/AugurProject/turbo/issues/310